src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -29,12 +29,11 @@ public class AddCommand extends Command {
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
-            + PREFIX_BIRTHDATE + "08-08-1988"
+            + PREFIX_BIRTHDATE + "08-08-1988 "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_TAG + "elderly ";
```

Comment:
This person won't be considered 'eldery', consider changing tag or changing birthdate